python/ray/rllib/agents/ppo/appo_policy_graph.py

```diff
@@ -1,488 +1,497 @@
-"""Adapted from VTracePolicyGraph to use the PPO surrogate loss.

```

Comment:
wait what's the change in this file?

Reply:
Somehow the github diff is messed up, but it's similar to the vtrace_policy_graph change.